### PR TITLE
Home

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

File: components/alert/index.html [hunk 1]
```diff
@@ -138,6 +180,33 @@ <h3>Detailed alert horizontal align</h3>
...
+          <a
+            href="https://www.linkedin.com/in/surbhi-kukreti-a91b0b163"
+            class="hui-footer-icon"
+          >
+            <i class="fab fa-linkedin"></i>
+          </a>
+        </li>
+      </ul>
+      <!-- <div id="legacyText" class="text-col-navy">made by Surbhi Kukreti</div> -->
+
```
Comment (Creator-singh) [inline components/alert/index.html]: remove if not needed

File: components/avatar/index.html [hunk 2]
```diff
@@ -199,5 +245,32 @@ <h3>6.Avatar circled with online badge</h3>
...
+          <a
+            href="https://www.linkedin.com/in/surbhi-kukreti-a91b0b163"
+            class="hui-footer-icon"
+          >
+            <i class="fab fa-linkedin"></i>
+          </a>
+        </li>
+      </ul>
+      <!-- <div id="legacyText" class="text-col-navy">made by Surbhi Kukreti</div> -->
+
```
Comment (Creator-singh) [inline components/avatar/index.html]: same here

**Review by aurobindoGupta — APPROVED** [review]
nice

File: components/badges/index.html [hunk 3]
```diff
@@ -6,70 +6,144 @@
...
+          <a
+            href="https://www.linkedin.com/in/surbhi-kukreti-a91b0b163"
+            class="hui-footer-icon"
+          >
+            <i class="fab fa-linkedin"></i>
+          </a>
+        </li>
+      </ul>
+      <!-- <div id="legacyText" class="text-col-navy">made by Surbhi Kukreti</div> -->
+
```
Comment (Creator-singh) [inline components/badges/index.html]: here as well

File: home-page/homepage.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.search-btn:hover {
+  transform: scale(1.05);
+}
+
+.comp-name {
+  text-decoration: none;
+  padding: var(--spacing-sm);
+  color: #04195b;
+  /* padding: var(--spacing-2xs); */
+}
```
Comment (Creator-singh) [inline home-page/homepage.css]: remove if not in use

**Review by Creator-singh — APPROVED** [review]
else looks good to me remove unwanted comments🚀

File: home-page/homepage.css (hunk 4 above)
Comment (surbhi84) [reply home-page/homepage.css]: done

**Pull request merged**